conda-recipes/qsharp/build.ps1

```diff
@@ -1,10 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# We need to set the PYTHON_VERSION environment variable
-# explicitly here, since conda-build doesn't by default pass
-# environment variables from the host environment.
-$Env:PYTHON_VERSION = $Env:PKG_VERSION
-Push-Location src/src/Python/
-    python setup.py install --prefix $Env:PREFIX
+Push-Location src/src/Python/dist
```

Comment:
If I understand right, this makes the `qsharp` conda package include all `qsharp-*` PyPI packages, rather than having a conda package for each PyPI package and then adding a `qsharp` conda metapackage?

Reply:
Yes, that was my train of thought, have a kitchen-sink conda package with all that we have, not like we have that much :P. I don't see much value of having fine-grain packages but I could be easily convinced otherwise.

Reply:
I think this approach makes sense. If we find a need for finer-grained conda packages down the road, we could always add them. But given that the Python libraries are small at the moment, I'm not sure there's a huge benefit there. One downside I guess is that the conda packages built from the CI builds in iqsharp will no longer contain the chemistry library, but I don't think that's a big deal, as one could manually install the `qsharp-chemistry` package if needed.

Reply:
Agreed. My only point of concern is the asymmetry between the two, such that the PyPI `qsharp-chemistry` package has no direct `conda` analogue. I think that can be addressed with documentation and by primarily promoting the `qsharp` PyPI metapackage.